webapp/src/Service/ImportExportService.php

```diff
@@ -309,8 +309,9 @@ public function importContestData(mixed $data, ?string &$errorMessage = null, st
      *              problems?: array{name?: string, short-name?: string, id?: string, label?: string,
      *                              letter?: string, label?: string, letter?: string}} $problems
      * @param string[]|null $ids
+     * @param array<string, string[]> $messages
      */
-    public function importProblemsData(Contest $contest, array $problems, array &$ids = null): bool
+    public function importProblemsData(Contest $contest, array $problems, array &$ids = null, ?array &$messages = []): bool
```

Comment:
```suggestion
    public function importProblemsData(Contest $contest, array $problems, array &$ids = null, ?array &$messages = []): true
```

Reply:
Well no, since it doesn't return true all the time anymore now?

Reply:
Missed that line indeed.